### ISSUE TITLE
Phone number

### ISSUE THREAD

**Issue by spham** [antonioribeiro/countries#5]
Hi
Can you include Phone format in E164 as feature ?

**Comment by antonioribeiro** [comment]
Yeah, would love to, but won't be able to until march 20, feel free to send a PR.

**Comment by uruloke** [comment]
@spham could you give an example on how you would like the data? 
Already making a pull request so shouldn't be a problem to add your request too.

The calling code is already included, so if I understand E164 correctly you just want to add a `+` before the calling code?

**Comment by spham** [comment]
something like this.
https://en.wikipedia.org/wiki/List_of_country_calling_codes#Alphabetical_listing_by_country_or_region

**Comment by uruloke** [comment]
@spham The only difference I see on this and the already included property `callingCode` is there is an `+` before. 
Is that correct?

**Comment by spham** [comment]
The recommandation are here, and seems a lots of complex than +
https://en.wikipedia.org/wiki/E.164

**Comment by uruloke** [comment]
My bad, I can see there is also a `National Destination Code` which is optional. 
Will try to see if I can find a dataset with the needed data.

**Comment by antonioribeiro** [comment]
@uruloke, did you found something?

**Comment by antonioribeiro** [comment]
National Destination Code, looks a lot like the telephone number. An example here in my country:

International calling phone number  = +5521980882233

+55 = Country Code (Brazil)
21980882233 = National Destination Code

21 = City Code (Rio de Janeiro)
980882233 = Local Phone Number

Local phone numbers can repeat on another city:

11 = City Code (São Paulo)
980882233 = Local Phone Number

That being said, take a look at those two datasets, please:

http://country.io/phone.json
https://github.com/datasets/country-codes/blob/master/data/country-codes.csv

In this last one I found some other data we are missing, like Geoname ID and EDGAR.

**Comment by uruloke** [comment]
@antonioribeiro http://country.io/phone.json This one seems a little inconsistence on when it wan't to use `+` at th beginning and when not to.
The last one looks a lot more clean and the data is at least correct for my country.

**Comment by uruloke** [comment]
@antonioribeiro my thoughts were more to integrate something like libphonenumber by google https://github.com/googlei18n/libphonenumber.
There is an PHP port from giggsey https://github.com/giggsey/libphonenumber-for-php, and also an Laravel port of giggseys port https://github.com/Propaganistas/Laravel-Phone.

**Comment by spham** [comment]
i think about new, issue 

how to store in database ? 

field  (prefix, phone )
or
field (phone e164 , i.e +337xxxxxxx )

**Comment by iBeriaLab** [comment]
country calling code display with [] 
["297"]
["93"]
["244"]
["1264"]
["358"]
["355"]
["376"]
["971"]
["54"]
["374"]
["1684"]......

please help me. how to remove [""] tag?
